paddle/fluid/operators/detail/brpc_client.cc

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/operators/detail/brpc_client.h"
+#include "paddle/fluid/framework/threadpool.h"
+
+namespace paddle {
+namespace operators {
+namespace detail {
+
+DEFINE_int32(brpc_channel_num, 24,
+             "Number of channels to send requests connected to one server");
+DEFINE_string(connection_type, "pooled",
+              "Connection type. Available values: single, pooled, short");
+DEFINE_int32(timeout_ms, -1, "RPC timeout in milliseconds");
```

Comment:
Since we have time_out specified in AsyncSendVar, it seems that we do not need this flag.

Reply:
Is there a way to apply timeout to one send or recv action instead of to channel options?

Reply:
Thanks!